docs/install.md

```diff
@@ -106,27 +126,38 @@ Once the module is installed and enabled, you can check its requirements on
 ![MongoDB on status page](images/mongodb-requirements.png)
 
 
-##Composer Requirements
+###Composer Requirements
 
-* Commands below are for those who are using composer already in your site to
-  manage module dependencies. To know more about composer [here][composer].
+* This section is applicable if you are using composer already in your site to
+  manage module dependencies. Know more about composer [here][composer].
 
 [composer]: https://www.drupal.org/docs/develop/using-composer/using-composer-to-manage-drupal-site-dependencies
 
-* At the root of your site
-  * If you are using the `drupal-composer/drupal-project` skeleton, just add
-    this package:
+  * At the root of your site
+    * If you are using the `drupal-composer/drupal-project` skeleton, just add
+      this package:
+
+         `composer require drupal/mongodb "^2.0.0"`
```

Comment:
Actually, until https://www.drupal.org/project/project_composer/issues/2985860 is fixed, this is apparently not working: adding the mongodb/mongodb requirement manually is still needed : this is already explained at the bottom of the file, but would probably be better if it was here instead.